_announcements/week-3.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Week 3 Announcement
+week: 3
+date: 2022-2-13
+---
+
+Hello everyone! Welcome to the Linux Sysadmin Decal! Here are your week 3 announcements:
+
+1. The third week’s labs and lectures have been released! You can view them below.
+2. Due to maintenance of OCF servers on Saturday, Lab 2 is now due Monday, 2/14 at 11:59 pm.
+3. Live lab sections are at 8pm Tuesday and Thursday! For instructions on how to navigate to the lab, see [ocf.io/lab](https://ocf.io/lab) **Due to high demand for the beginner track, we are opening up Thursday sections to beginner students as well.**
+4. In the unlikely event that the physical lab capacity is exceeded, in-person labs will be **first come, first served** with priority given to officially enrolled students.
```

Comment:
i think we can remove the fcfs announcement, lab capacity doesn't seem to be an issue